src/clients/api/queries/getMintableVai/types.ts

```diff
@@ -0,0 +1,12 @@
+import BigNumber from 'bignumber.js';
+
+import { VaiUnitroller } from 'types/contracts';
+
+export interface GetMintableVaiInput {
+  vaiControllerContract: VaiUnitroller;
+  accountAddress: string;
+}
+
+export type GetMintableVaiOutput = {
```

Comment:
nit: observing that previously we would just return the value without wrapping it in an object

Reply:
Also should be an interface

Reply:
We did and there's actually two issues with that:
- we can't know what unit the resulting amount is in (unless we'd rename the function, which we've done in a few instances but not most)
- if this query was to change and more parameters would be added, it would be simpler to extend the object returned than to transform the return type from a BigNumber to an object

Let me know your thoughts; I think it could be nice to implement in the codebase.

Reply:
It does need to be an interface yeah

Reply:
> We did and there's actually two issues with that:
> 
> * we can't know what unit the resulting amount is in (unless we'd rename the function, which we've done in a few instances but not most)
> * if this query was to change and more parameters would be added, it would be simpler to extend the object returned than to transform the return type from a BigNumber to an object
> 
> Let me know your thoughts; I think it could be nice to implement in the codebase.

works for me. Sounds nicer

Reply:
Ok great, I'll create a ticket to update the other queries and mutations